lib/Trolley/OfflinePayment.php

```diff
@@ -78,6 +102,22 @@ public static function search($params = [])
         return Configuration::gateway()->offlinePayments()->search($params);
     }
 
+    /**
+	 * Returns all Offline Payments.
```

Comment:
Format werid

Reply:
Sorry, which format are we referring to?